src/main/java/server/router/RequestRouter.java

```diff
@@ -0,0 +1,36 @@
+package server.router;
+
+import java.util.Arrays;
+import java.util.List;
+import server.request.Request;
+import server.response.Response;
+import server.router.routes.HeadRequest;
+import server.router.routes.NotFound;
+import server.router.routes.Route;
+import server.router.routes.SimpleGet;
+import server.router.routes.SimpleGetWithBody;
+
+public class RequestRouter {
+  private Request request;
+  private List<Route> routes;
+
+  public Response getResponse(Request request) {
+    this.request = request;
+    routes = getRouteList();
+
+    for (Route route : routes) {
+      if (request.path().equals(route.path())) {
+        return route.processRequest();
+      }
+    }
+    return new NotFound(request).processRequest();
+  }
+
+  private List<Route> getRouteList() {
```

Comment:
Tiny nitpick - this method name seems to be revealing too much about its implementation. If we wanted to use a different data structure other than a `List`, we'd have to change the name of this method. Can you think of another name for this method?